applications/nrf5340_audio/doc/firmware_architecture.rst

```diff
@@ -122,31 +124,22 @@ These modules include the following major ones:
 
 * Application-specific custom modules:
 
-  * Stream Control - This module handles events from the Bluetooth modules and buttons, receives audio from one module, and forwards the audio data to the next module.
-
-    * Currently, each of the four main device types uses a separate stream control file:
-
-      *  CIS gateway (unicast client) - :file:`streamctrl_unicast_client.c`
-      *  CIS headset (unicast server) - :file:`streamctrl_unicast_server.c`
-      *  BIS gateway (broadcast source) - :file:`streamctrl_broadcast_source.c`
-      *  BIS headset (broadcast sink) - :file:`streamctrl_broadcast_sink.c`
-
   * FIFO buffers
   * Synchronization module (part of `I2S-based firmware for gateway and headsets`_) - See `Synchronization module overview`_ for more information.
 
-Since the application architecture is uniform and the firmware code is shared, the set of audio modules in use depends on the chosen stream mode (BIS or CIS), the chosen audio inputs and outputs (USB or analog jack), and if the gateway or the headset configuration is selected.
+Since the application architecture is the same for all applications and the code before compilation is shared to a significant degree, the set of modules in use depends on the chosen audio inputs and outputs (USB or analog jack).
```

Comment:
Above: Suggest we remove the list of custom modules.

Reply:
Would keeping the synchronization module acceptable?